sdk/python/feast/offline_server.py

```diff
@@ -17,10 +17,10 @@
 from feast.permissions.server.arrow import (
     arrowflight_middleware,
     inject_user_details,
+    inject_user_details_decorator,
```

Comment:
maybe the `decorator` part is redundant in the name

Reply:
any thought on function name then ?

Reply:
Oh I see, there's already a function with that name. Let's keep the suffix then.